The role of E2F regulation and H3K79 methylation in Oikopleura dioica's cell cycle modes
=============

André Figueiredo Rendeiro, 2014

Work performed at the Sars Centre for Marine Molecular Biology, Bergen, Norway

Thesis for the master in Molecular and Cellular Biology, University of Aveiro, Portugal

# Abstract
The eukaryotic cell cycle is one of the most studied biological processes. However, variations of the canonical cell cycle have been discovered and found to be more predominant than previously expected. Endoreplication and endocycling - two such variants - produce polyploid cells, conferring advantages in growth and genotoxic stress. Although some of the regulatory principles of these cycles are being discovered, little is known about how cells can transition from a mitotic cell cycle to them.

Recently, it has been proposed that methylation of histone 3 lysine 79 (H3K79me) might have a function distinct from most histone modifications - a timer for a cell's age -, for methylation of all H3K79 states is performed in a distributive fashion (unlike all other methylated lysines in histones) by Dot1 and no demethylase is known. Functional studies on Dot1 have shown aberrant patterns of replication, linking it directly to a cell's ability to regulate its cell cycle.

This project aims to study the mechanisms of transition between different cell cycle modes in *Oikopleura dioica* - a marine chordate employing endocycling extensively through its life cycle.
We performed ChIP-seq on two antagonist transcription factors (E2F1 and E2F7) involved in the control of cell cycle progression that have been shown to have differential patterns of regulation in mitotic and endocycling cell cycle modes. Results reveal a high degree of spatial and temporal occupancy of both transcription factors in close proximity to transcription start sites, regulating genes with a function highly enriched for the control of replication and cell cycle. We set ourselves to perform an initial functional study of H3K79me in *Oikopleura* though Dot1 inhibition in developmental stages employing different cell cycle modes. This caused deregulation of proliferation patterns in different ways in the different cell cycle modes, highlighting a contribution of H3K79me to their regulation in *Oikopleura*.

# Table of contents

1. Introduction
	1. Polyploid cell cycles - endoreplication and endocycles
		1. An overview of the eukaryotic canonical cell cycle
			1. Regulation of cell cycle progression by Cyclin-CDK complexes
		2. Endocycle regulation
			1. Abolishment of mitosis and cell division
		2. Alternative regulation of cell cycle drivers
			1. Endoreplication by CDK activity regulation
			2. E2F factors in CDK activity regulation
	2. Histone modifications in cell cycle regulation
		1. The role of H2Bub in DNA replication
		2. Dot1 and H3K79 methylation in the cell cycle
	3. Oikopleura dioica, a marine chordate extensively employing endocycles
		1. Oikopleura’s life cycle
		2. The genome of Oikopleura dioica
		3. Histone variants and modifications of Oikopleura dioica
		4. The cell cycle modes of Oikopleura dioica
		5. Growth arrest
	4. Project Goals

2. Materials and Methods
	1. Materials
		1. Antibodies
		2. Chemicals and reagents
		3. Consumables
		4. Instruments and equipment
		5. Buffers and solutions
			1. ChIP
			2. Western blot
			3. Immunostaining
	2. Animal culture and collection
		1.	Culture of O. dioica
		2. Collection of O. dioica
		 	1. Tailbud stage
			2. Day two stage
			3. Day six, immature stage
	3. ChIP-seq
		1. Animal fixation
		2. Cell lysis and chromatin sonication
		3. Chromatin quality assessment
		4. Immunoprecipitation
		5. qPCR
		6. Illumina library construction and high-throughput sequencing
	4. Western blot
		1. Sample preparation
		2. SDS-PAGE
		3. Protein transference
		4. Blotting
		5. Detection
	5. Whole mount immunostaining
	6. Dot1L inhibition
	7. Tunel assay
	8. EdU incorporation and detection
	9. Data analysis.
		1. Finding of ChIP-enriched regions
		2. Genomic characterization of ChIP-enriched regions.
		3. Transcription factor motif finding in ChIP-enriched regions
		4. Characterization of the epigenetic landscape of ChIP-enriched
		5. ChIP-enriched regions and gene association
		6. Detection of significant Gene Ontology terms
		7. Detection of the expression levels of genes

3. Results
	1. E2F factors in Oikopleura’s late developmental stages
		1. Detection of E2F binding through ChIP-chip
			1. Validation of E2F antibodies
			2. Genomic context of E2F binding sites
			3. Insights into the mechanism of E2F binding
			4. Spatial and temporal co-localization of E2F factors
		2. Putative E2F regulated genes
	2. Dot1 and H3K79 methylation in Oikopleura dioica’s cell cycle modes

4. Discussion
	1. E2F factors in Oikopleura’s late developmental stages
	2. Dot1 and H3K79 methylation in Oikopleura dioica’s cell cycle modes

5. Conclusions and Future Perspectives
